### PR TITLE
Group extension point declaration + package per language

### PR DESCRIPTION
**Pull request by angelozerr**
This PR groups extension point declaration + package per language. It's the first step before splitting plugins per language https://github.com/mickaelistria/eclipse-bluesky/issues/26

### CODE REVIEW

File: org.eclipse.bluesky/plugin.xml [hunk 1]
```diff
@@ -1,25 +1,88 @@
...
+            class="org.eclipse.tm4e.ui.text.TMPresentationReconciler"
+            contentType="org.eclipse.bluesky.parent">
+      </presentationReconciler>
+   </extension>
+   
+   <extension
+         point="org.eclipse.ui.handlers">
+      <handler
+            class="org.eclipse.bluesky.debug.ToogleLineBreakpointHandler"
+            commandId="org.eclipse.debug.ui.commands.ToggleLineBreakpoint">
```
Comment (mickaelistria) [inline org.eclipse.bluesky/plugin.xml]: This one is only for JS files so far (although the `enableWhen` is missing), so you should consider moving this to the JS block.

**Comment by angelozerr** [comment]
@mickaelistria is there some other things that you wish I fix? I would like to create a next PR with language configuration.

**Comment by mickaelistria** [comment]
Can you please squash the commits in a single one and rebase it on top of master?

**Comment by angelozerr** [comment]
See my new PR at https://github.com/mickaelistria/eclipse-bluesky/pull/32

**Comment by mickaelistria** [comment]
FYI, you can replace a series of commits on a PR by a single one by squashing locally and then using `git push --force yourFork HEAD:branch-from-which-pr-was-made` to prevent for multiple PRs.

**Comment by angelozerr** [comment]
I'm so noob with git -( I have tried it but it didn't work. Thanks for your information but I have deleted my forked repo that's why U have created a new PR #32 Please accept it, thanks!

**Pull request closed without merging**